src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.Unix.cs

```diff
@@ -167,7 +167,7 @@ private void CopyFromScreenX11(int sourceX, int sourceY, int destinationX, int d
             /* Get XVisualInfo for this visual */
             visual.visualid = LibX11Functions.XVisualIDFromVisual(defvisual);
             vPtr = LibX11Functions.XGetVisualInfo(Gdip.Display, 0x1 /* VisualIDMask */, ref visual, ref nitems);
-            visual = (XVisualInfo)Marshal.PtrToStructure(vPtr, typeof(XVisualInfo))!;
+            visual = Marshal.PtrToStructure<XVisualInfo>(vPtr)!;
```

Comment:
The plan is to delete Unix support for System.Drawing.Common in .NET 7: https://github.com/dotnet/designs/blob/main/accepted/2021/system-drawing-win-only/system-drawing-win-only.md

So there is not much value in fixing the Unix specific files.

Reply:
Should I remove these changes or I can leave them as is? I would not be offended if that's get removed. This is mindless changes.

Reply:
It is ok to keep it.